weave/serialize2.py

```diff
@@ -0,0 +1,233 @@
+from typing import Any, Dict, Tuple, Union, TypedDict, MutableMapping
```

Comment:
Same here: would be helpful to have comment in the header about how this differs from serialize

Reply:
Dunzo